pom.xml

```diff
@@ -162,8 +157,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.2</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
I have concerns with bumping this to 1.8 immediately. But, meh, lets do it, If this becomes too big of a problem, we can always bring in something like Eclipse Collections, my personal favorite collections library and revert to java 1.7. But I suspect 1.8 bump will be ok.